src/person_linkage_case_study_utils/start_spark_workers.py

```diff
@@ -0,0 +1,54 @@
+#!/bin/env python
+
+import os
+import socket
+import subprocess
+from argparse import ArgumentParser
+from pathlib import Path
+
+parser = ArgumentParser()
+parser.add_argument("singularity_path")
+parser.add_argument("singularity_image")
+parser.add_argument("singularity_args")
+parser.add_argument("conda_path")
+parser.add_argument("conda_prefix")
+parser.add_argument("venv_path")
+parser.add_argument("master_url")
+parser.add_argument("local_dir")
+
+args = parser.parse_args()
+
+host = socket.getfqdn()
+
+if args.conda_path != "":
```

Comment:
could you have the same `"None"` situation as you check for in `spark_master.py`?

Reply:
Yes! Good catch!